spec/interactors/create_user_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe CreateUser, type: :service do
```

Comment:
@LucasAnderson1984 I would recommend type of `:interactor`, my apologies if the example was `:service`... I may of been feeling my way. :smile: